src/components/input/IssueFilterDropdowns.vue

```diff
@@ -13,10 +13,10 @@
             :item-manager="itemManager"
             :mapper="(item) => item.labels.nodes"
             :additional-initial-values="
-                async () =>
```

Comment:
maybe move that whole function down to the script part, this is a bit long for the template